games/supertuxkart/files/patch-supertuxkart-iosurfaceref.diff

```diff
@@ -0,0 +1,13 @@
+https://trac.macports.org/ticket/67263
+
+--- lib/graphics_engine/include/vk_mvk_moltenvk.h.orig	2023-05-14 14:43:03
++++ lib/graphics_engine/include/vk_mvk_moltenvk.h	2023-05-14 14:43:21
+@@ -26,7 +26,7 @@
+ extern "C" {
+ #endif	//  __cplusplus
+ 	
+-#include <IOSurface/IOSurfaceRef.h>
++typedef void *IOSurfaceRef;
+ 
+ #ifdef __OBJC__
+ #import <Metal/Metal.h>
```

Comment:
Presumably we also need to disable exclusion of Metal here too?

Reply:
This actually works out OK – `__OBJC__` is defined in only one file where this header is included, so the provided undef patch takes care of it.

Reply:
Never Mind, didn't expect you to respond so fast! LOL

I'll let you run with this then, you know it better than I do.

Reply:
Oh – I see what you mean. IOSurfaceRef is indirectly included by Metal. Hmm...